Classes/Indexer/NodeIndexer.php

```diff
@@ -135,8 +135,7 @@ public function indexNode(NodeInterface $node, $targetWorkspaceName = null, $ind
 
         $fulltextData = [];
 
-        if (isset($this->indexedNodeData[$identifier])) {
-            $properties = $this->indexClient->findOneByIdentifier($identifier);
+        if (isset($this->indexedNodeData[$identifier]) && ($properties = $this->indexClient->findOneByIdentifier($identifier)) !== false) {
```

Comment:
i see `findOneByIdentifier` returns false on failure ... (the method is not defined in the interface btw but whatever ^^ https://github.com/Flowpack/Flowpack.SimpleSearch/blob/6a8c1f84cce592afe0ca7587f73e65ee1a80e7d5/Classes/Domain/Service/MysqlIndex.php#L222)

and because php converts false to array if necessary the error is indeed `Warning: Undefined array key "__workspace"`